cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wethegit/corgi",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
I don't think the YAML updates are likely to be at all backwards compatible here, are they? If not, we should look at updating this to a 2.0.0.

Reply:
Good point Liam, I don't think it will be indeed.

Reply:
Updated PR title to `v2.0.0`.